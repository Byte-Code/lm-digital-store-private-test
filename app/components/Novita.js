import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  height: 60px;
  position: relative;
  z-index: 2;
`;

const TopLabel = styled.div`
  width: 0;
  height: 0;
  border-style: solid;
  border-width: 60px 60px 0 0;
  border-color: #6699cc transparent transparent transparent;
  position: absolute;
  top: 0;
  left: 0;
`;

const Text = styled.span`
 position: absolute;
 width: 85px;
 height: 38px;
 top: -5px;
 left: -26px;
 line-height: 65px;
 text-align: center;
 color: #ffffff;
 text-transform: uppercase;
 font-size: 12px;
 font-family: "LeroyMerlinSansBold", sans-serif;
 -webkit-transform: rotate(316deg); //fai 315deg
 -moz-transform: rotate(316deg);
 -o-transform: rotate(316deg);
 writing-mode: lr-tb;
`;

export default function Icon() {
  return (
    <Wrapper>
      <TopLabel />
      <Text>NOVITÃ€</Text>
    </Wrapper>
  );
}
