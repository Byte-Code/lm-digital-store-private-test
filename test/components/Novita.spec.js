import React from 'react';
import { shallow } from 'enzyme';

import Novita from '../../app/components/Novita';

describe('Novita', () => {
  it('should render properly', () => {
    const result = shallow(<Novita />);
    expect(result).toMatchSnapshot();
  });
});
