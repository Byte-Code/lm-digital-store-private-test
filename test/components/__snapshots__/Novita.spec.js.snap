// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Novita should render properly 1`] = `
<styled.div>
  <styled.div />
  <styled.span>
    NOVITÃ€
  </styled.span>
</styled.div>
`;
